web/src/lib/components/Navbar.svelte

```diff
@@ -72,64 +110,99 @@
     onDestroy(unsubscribe);
 </script>
 
-<div class="navbar bg-base-300">
-    <div>
-        <a href={getBaseUrl()}>
-            <img src="/images/logo.jpg" class="mr-3 h-6 sm:h-9 rounded" alt="Plebeian Technology" />
+<nav class="navbar flex items-center bg-base-300 flex-wrap">
+    <a href={getBaseUrl()} class="flex" on:click={closeMenu}>
+        <img src="/images/logo.jpg" class="w-10 h-10 rounded align-middle" alt="Plebeian Technology" />
+        <span class="md:hidden inline-block card-title text-2xl items-center text-primary ml-2">
+            Plebeian Market
+        </span>
+        {#if isLocal() || isStaging() }
+            <!-- hiddin on small devices -->
+            <div class="hidden md:inline badge badge-primary ml-2">{getEnvironmentInfo()}</div>
+        {/if}
+    </a>
+    <div class="hidden md:block">
+        <a class="rounded btn btn-ghost normal-case text-xl" href={null} on:click|preventDefault={() => goto("/")}>
```

Comment:
any reason to use goto instead of href? this is a link, after all... feels non-intuitive to have a link without href. this would also break "open in new window/tab"